src/lib/components/compositions/checkbox/animated.story.svelte

```diff
@@ -38,11 +38,11 @@
 </script>
 
 {#if withForm}
-  <Form onSubmitWhenValid={submitForm}>
+  <Form ariaDescribedBy="animatedCheckboxStory" onSubmitWhenValid={submitForm}>
     <Checkbox
       bind:this={el}
       {checkValidity}
-      firstInvalidInput={false}
+      isFirstInvalidInput={false}
```

Comment:
You'll see a lot of this. I'm trying to normalize the boolean metadata variables to start with `is` for consistency.